Dockerfile

```diff
@@ -3,9 +3,9 @@ FROM golang:1.18 as build
 WORKDIR /go/src/app
 COPY . .
 
-RUN go mod download
-RUN CGO_ENABLED=0 make install
-
-FROM gcr.io/distroless/static-debian11
+RUN go mod download \
+  && CGO_ENABLED=0 make install
```

Comment:
These were done on seperate layers on purpose, mind seperating them back out?

Reply:
Are the layers separated because of caching?

Reply:
Yeah exactly - sorry I should have mentioned, the notion is that we maybe don't often always update libs and code, so we might get some speed out of having modules cached seperarly.

Reply:
No problem, also I check your GitHub Actions workflow that publish the container image and the actions build and push is not using cache. Do you mind if I open a PR to add that?

Reply:
>Do you mind if I open a PR to add that?

Not at all!